setup.py

```diff
@@ -31,6 +31,8 @@
 
 install_requires = [
     'BeautifulSoup>=3.2.1',
+    # sentry also requires this, so we're just enforcing that it needs to exist
+    'boto3>=1.4.4,<1.5.0',
```

Comment:
something was broken on 1.4.1 IIRC (it might have been the messageduplicationid)